src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -63,9 +61,20 @@ public void TestRootGetProcessById()
             Assert.Equal(1, p.Id);
         }
 
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Linux)]
+        public void ProcessStart_UseShellExecute_OnLinux_ThrowsIfNoProgramInstalled()
+        {
+            if (!_allowedProgramsToRun.Any(program => IsProgramInstalled(program)))
+            {
+                Console.WriteLine($"None of the following programs were installed on this machine: {string.Join(",", _allowedProgramsToRun)}.");
+                Win32Exception e = Assert.Throws<Win32Exception>(() => Process.Start(new ProcessStartInfo { UseShellExecute = true, FileName = Environment.CurrentDirectory }));
```

Comment:
Nit: we're not using the result of the Throws, so we don't need to store it into a local